.github/workflows/build_and_test.yml

```diff
@@ -1123,13 +1126,15 @@ jobs:
               -f deployProxyAuthBroker=true \
               -f branch=main \
               -f dockerTag=${{needs.tag.outputs.build-tag}}
+          elif [[ "$GITHUB_REF_NAME" == "release-compute" ]]; then
+            echo Start here a workflow for the deployment of the compute release
```

Comment:
We don't have a workflow for that yet, so I would rephrase it to avoid confusing anyone.